ddtrace/internal/flare.py

```diff
@@ -47,90 +66,66 @@ def prepare(self, configs: List[dict]):
             except Exception as e:
                 log.error("Failed to create %s directory: %s", TRACER_FLARE_DIRECTORY, e)
                 return
-        for agent_config in configs:
-            # AGENT_CONFIG is currently being used for multiple purposes
-            # We only want to prepare for a tracer flare if the config name
-            # starts with 'flare-log-level'
-            if not agent_config.get("name", "").startswith("flare-log-level"):
-                return
 
-            # Validate the flare log level
-            flare_log_level = agent_config.get("config", {}).get("log_level").upper()
-            flare_log_level_int = logging.getLevelName(flare_log_level)
-            if type(flare_log_level_int) != int:
-                raise TypeError("Invalid log level provided: %s", flare_log_level_int)
-
-            ddlogger = get_logger("ddtrace")
-            pid = os.getpid()
-            flare_file_path = TRACER_FLARE_DIRECTORY / pathlib.Path(f"tracer_python_{pid}.log")
-            self.original_log_level = ddlogger.level
-
-            # Set the logger level to the more verbose between original and flare
-            # We do this valid_original_level check because if the log level is NOTSET, the value is 0
-            # which is the minimum value. In this case, we just want to use the flare level, but still
-            # retain the original state as NOTSET/0
-            valid_original_level = 100 if self.original_log_level == 0 else self.original_log_level
-            logger_level = min(valid_original_level, flare_log_level_int)
-            ddlogger.setLevel(logger_level)
-            self.file_handler = _add_file_handler(
-                ddlogger, flare_file_path.__str__(), flare_log_level, TRACER_FLARE_FILE_HANDLER_NAME
-            )
-
-            # Create and add config file
-            self._generate_config_file(pid)
-
-    def send(self, configs: List[Any]):
+        flare_log_level = flare_prep_req.log_level
+
+        ddlogger = get_logger("ddtrace")
+        pid = os.getpid()
+        flare_file_path = TRACER_FLARE_DIRECTORY / pathlib.Path(f"tracer_python_{pid}.log")
+        self.original_log_level = ddlogger.level
+
+        # Set the logger level to the more verbose between original and flare
+        # We do this valid_original_level check because if the log level is NOTSET, the value is 0
+        # which is the minimum value. In this case, we just want to use the flare level, but still
+        # retain the original state as NOTSET/0
+        valid_original_level = 100 if self.original_log_level == 0 else self.original_log_level
```

Comment:
what is `100`?

It might be better to use `logging.NOTSET` / `logging.CRITICAL` / etc instead of the numeric values so it is easier to read.

Reply:
Good call, I'll fix these